src/Badge/Model/UseCase/CreateComposerLockBadge.php

```diff
@@ -55,13 +55,13 @@ public function __construct(
      * @throws UnexpectedValueException
      * @throws \GuzzleHttp\Exception\GuzzleException
      */
-    public function createComposerLockBadge(string $repository, string $format = 'svg'): CacheableBadge
+    public function createComposerLockBadge(string $repository, string $format = 'svg', string $style = 'flat'): CacheableBadge
```

Comment:
We can use a constant for `flat` or other values?

Reply:
We could use `App\Badge\Model\Badge::DEFAULT_STYLE`. I didn't to keep the signatures consistent. If we refactor this, I would suggest also replacing the default value of `$format` with `App\Badge\Model\Badge::DEFAULT_FORMAT`

Reply:
I agree @ricardoboss